packages/webpack-config/webpack/webpack.prod.js

```diff
@@ -57,12 +60,14 @@ module.exports = async function(env = {}, argv) {
               // https://github.com/terser-js/terser/issues/120
               inline: 2,
             },
-            mangle: {
-              safari10: true,
-            },
+            mangle: isDebugMode
+              ? false
+              : {
```

Comment:
seems like we should always pass the safari 10 option so that it works in safari 10. or does passing in any options cause terser to run?

Reply:
any option counts as `true`